projects/addon-doc/src/components/example/example.component.ts

```diff
@@ -28,10 +29,10 @@ import {TUI_DOC_EXAMPLE_TEXTS} from '../../tokens/i18n';
 })
 export class TuiDocExampleComponent {
     @Input()
-    heading: string | null = null;
+    heading: PolymorpheusContent | null = null;
```

Comment:
How about we drop `null` and default to empty string? I don't think people would notice